app/sass/_base.scss

```diff
@@ -0,0 +1,62 @@
+// import google font
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&family=Almarai&family=K2D:wght@500&family=Kodchasan&family=Mulish:wght@500;600;700;800;900;1000&display=swap");
+
+body,
+html {
+  overflow-x: hidden;
```

Comment:
Domyślam się że pojawiło się to dlatego że pojawił sie pasek przewijania na dole. Jeżeli tak, to raczej trzeba iść w kierunku naprawienia tego co powoduje że ten pasek się pojawia. W devtoolsach można usuwać poszczegolne sekcje z htmla, ten sposob moze ułatwić debugowanie. Kiedy po usunięciu jakies sekcji zniknie psaek - wtedy wiadomo że to tam coś nie tak

Reply:
Hmm to dodałem tworząc pierwszego SASSa, wzorowałem się na innych :)